packages/compiler-core/__tests__/compile.spec.ts

```diff
@@ -45,11 +47,9 @@ describe('compiler: integration tests', () => {
   }
 
   test('function mode', () => {
-    const { code, map } = compile(source, {
-      sourceMap: true,
-      filename: `foo.vue`
-    })
-
+    const config = { sourceMap: true, filename: `foo.vue` }
+    const { code, map } = compile(source, config)
```

Comment:
> The PR should fix the intended bug only and not introduce unrelated changes.

I think it should be reverted.

Reply:
But my add test requires the same configuration.
compile.spec.ts
```javascript
  // ...
  const config = { sourceMap: true, filename: `foo.vue` }
  const { code, map } = compile(source, { sourceMap: true, filename: `foo.vue` })
  // ...
  expect(compile(comp1, config).code).toEqual(compile(comp2, config).code)
```
So. should it be restored?